explorer/interpreter/action.h

```diff
@@ -27,11 +27,7 @@ namespace Carbon {
 // not compile-time constants.
 class RuntimeScope {
  public:
-  enum class State {
-    Normal = 0,
-    Destructor = 1,
-    CleanUpped = 2,
-  };
+  enum class State { Normal = 0, Destructor = 1, CleanUpped = 2, Method = 4 };
```

Comment:
Do these enum values need to have numbers assigned?

Reply:
Done!